onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -129,6 +130,9 @@ InlinedVector<std::unique_ptr<RewriteRule>> GenerateRewriteRules(
       rules.push_back(std::make_unique<ConvBNFusion>());
       rules.push_back(std::make_unique<ClipQuantFusion>());
       rules.push_back(std::make_unique<ReluQuantFusion>());
+#ifdef ENABLE_TRAINING
+      rules.push_back(std::make_unique<PythonOpPriorityRewriter>());
```

Comment:
should this even be here? You are already including it in the graph_transformer_utils.cc under orttraining. Even if we do want to add this here, it needs to be moved under "#ifdef ENABLE_TRAINING_TORCH_INTEROP" macro

Reply:
I looked at CI failures and this is causing the failures.

Reply:
Thanks I removed the node priority change from this PR. will have it together with other perf fixes later.